db/data.js

```diff
@@ -896,9 +889,9 @@ const produce = [
   {
     foodName: "Orange",
     imageURL:
-      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637899/Seasonal/057-organic-food_pikcaf.png",
+      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1666175321/Seasonal/021-orange_oafni2.png",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed feugiat euismod augue, ac fringilla diam pharetra id. In porttitor convallis nisi vel finibus. Pellentesque at diam venenatis, ornare mi eget, vestibulum arcu. Vivamus consectetur, erat eu semper dapibus, dolor sem suscipit mi, ut dignissim enim nibh vel risus. Sed aliquet, sapien eu congue gravida, lorem metus bibendum mauris, quis maximus nibh erat eu erat. Cras vulputate purus magna, tincidunt porttitor massa vestibulum sed. Nam turpis massa, dictum eu accumsan in, posuere sit amet lorem. Phasellus est sapien, faucibus eget elit et, blandit consequat ex. Proin feugiat arcu a magna iaculis, nec pharetra orci porta. Proin vel orci ante. Aliquam viverra non ligula nec rhoncus. Ut et tincidunt elit. Phasellus non risus in mi vulputate venenatis vitae quis enim. Vestibulum quis dolor venenatis, lobortis diam in, faucibus turpis. Nullam quis pharetra nulla, vestibulum tincidunt ipsum. Donec non ultrices ligula. Quisque at interdum velit. Donec laoreet elit enim, a vulputate risus cursus eu. Duis et est in mi pulvinar rhoncus in ut lorem. Integer nec dolor et velit sollicitudin vehicula. Duis sapien justo, condimentum nec orci eu, dictum dapibus nisl. Praesent ultricies libero vel leo pulvinar molestie.",
+      "An iconic citrus fruit, which sometimes comes in varieties which are not actually orange-coloured at all. Oranges can be sweet or bitter, and can be tinged with yellow or red. As the various types of oranges ripen at different times, different oranges are available year-round in the UK, though they do all need to be imported, as the UK does not have the correct climate for growing oranges at scale. Oranges are a good source of vitamins and antioxidants, especially vitamin C and citric acid, which are important compounds for encouraging the absorption of iron. Oranges are often eaten raw as a snack, though they can also be turned into juices, added to drinks, or used in fruit salads or to make sauces and marmalades. They are a good addition to the pan when cooking game and fish too. Citrus plants are not hardy in the UK, though you may have some success growing orange trees inside of a greenhouse in particularly sunny spots.",
```

Comment:
Take out the "or" before used in fruit salads to keep the list format? i.e. instead of "Oranges are often eaten raw as a snack, though they can also be turned into juices, added to drinks, or used in fruit salads or to make sauces and marmalades." have "Oranges are often eaten raw as a snack, though they can also be turned into juices, added to drinks, used in fruit salads or to make sauces and marmalades."

Reply:
Aye aye, el capitano.